### ISSUE TITLE
About "Minimum viable product"

### ISSUE THREAD

**Issue by ItsExtra** [ItsExtra/m4ftm#37]
[Wikipedia](https://en.wikipedia.org/wiki/Minimum_viable_product):

> A minimum viable product has just enough core features to effectively deploy the product, and no more. Developers typically deploy the product to a subset of possible customers—such as early adopters thought to be more forgiving, more likely to give feedback, and able to grasp a product vision from an early prototype or marketing information.

[Wikipedia](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar) on The Cathedral and the Bazaar: Musings on Linux and Open Source by an Accidental Revolutionary

>    Lessons for creating good open source software
> 1.  Every good work of software starts by scratching a developer's personal itch.
> 1.  Good programmers know what to write. Great ones know what to rewrite (and reuse).
> 1.  Plan to throw one [version] away; you will, anyhow. (Copied from Frederick Brooks' The Mythical Man-Month)
> 1.  If you have the right attitude, interesting problems will find you.
> 1.  When you lose interest in a program, your last duty to it is to hand it off to a competent successor.
> 1.  Treating your users as co-developers is your least-hassle route to rapid code improvement and effective debugging.
> 1.  Release early. Release often. And listen to your customers.
> 1.  Given a large enough beta-tester and co-developer base, almost every problem will be characterized quickly and the fix obvious to someone.
> 1.  Smart data structures and dumb code works a lot better than the other way around.
> 1.  If you treat your beta-testers as if they're your most valuable resource, they will respond by becoming your most valuable resource.
> 1.  The next best thing to having good ideas is recognizing good ideas from your users. Sometimes the latter is better.
> 1.  Often, the most striking and innovative solutions come from realizing that your concept of the problem was wrong.
> 1.  Perfection (in design) is achieved not when there is nothing more to add, but rather when there is nothing more to take away. (Attributed to Antoine de Saint-Exupéry)
> 1.  Any tool should be useful in the expected way, but a truly great tool lends itself to uses you never expected.
> 1.  When writing gateway software of any kind, take pains to disturb the data stream as little as possible—and never throw away information unless the recipient forces you to!
> 1.  When your language is nowhere near Turing-complete, syntactic sugar can be your friend.
> 1.  A security system is only as secure as its secret. Beware of pseudo-secrets.
> 1.  To solve an interesting problem, start by finding a problem that is interesting to you.
> 1.  Provided the development coordinator has a communications medium at least as good as the Internet, and knows how to lead without coercion, many heads are inevitably better than one.

Note: The author of the above is a [fucking lunatic](https://en.wikipedia.org/wiki/Eric_S._Raymond#Political_beliefs_and_activism). 

------

[DDG search](https://duckduckgo.com/?q=Minimum+viable+product+template+website&t=ffab&ia=web)

[Cheat Sheet: Minimum Viable Product](https://grasshopperherder.com/cheat-sheets/cheat-sheet-minimum-viable-product/)

> Skill Goal:     Be able to test a product hypothesis with minimal resources

>    Who is our customer?
    What are their pains?
    What job needs to be done?
    Is our customer segment too broad?
    How do we find them?

**Comment by ItsExtra** [comment]
https://grasshopperherder.com/what-a-journalist-can-teach-lean-startups-about-customer-interviews

> a customer discovery interview is a conversation, not a survey or a checklist. And conversations are all about shared experiences — finding commonalities that let us trade stories and riff on each others’ details.

> LISTEN TO WHAT THEY ARE NOT SAYING

It sounds more like a zen koan than practical advice, but there is real, quantifiable value in paying attention to what is absent as much as what is present. Musicians call it listening to the notes that aren’t being played. Joel and Ethan Coen have built their filmmaking careers on the quiet moments between the dialogue. And when someone is telling us a story, every detail they leave out hides a story of its own.

**Comment by ItsExtra** [comment]
[Planning a Minimum Viable Product: a Step-By-Step Guide](https://medium.com/@ClrMobile/planning-a-minimum-viable-product-a-step-by-step-guide-6f387d657870)

> TL;DR: Planning Your Minimum Viable Product
> 1. Identify and Understand The Business Needs
> a) Determine the long-term goal of the product and write it down
> b) Answer the question “Why are we doing this project?”
> c) Identify the success criteria that will indicate whether or not the product is successful
> 
> 2. Find The Opportunities
> a) Map out the user journey(s)
> Identify the users (actors)
> Identify the story ending (end goal)
> Identify all actions the user must take to meet that end goal
> b) Create a “pain and gain” map for each action
> Write down the action the user completes when using the product
> Write down the pain points for each action
> Write down the gains for each action
> c) Summarize the pains and gains into opportunity statements
> Use “How might we” statements or a similar method to summarize the pains and gains you have identified
> 
> 3. Decide What Features To Build
> a) Use opportunity statements to finalize your features
> b) Provide a breakdown of the features to include in the product roadmap
> c) Use a prioritization matrix (or similar method) to prioritize features

**Comment by ItsExtra** [comment]
timogoebel/dm-howto-open-source: [Thoughts on how we want to open-source and contribute to the community ](https://github.com/timogoebel/dm-howto-open-source)

I'm not sure what exactly this is about. Some company I guess. But the test is useful and has some interesting looking links.

**Comment by ItsExtra** [comment]
Tags on Github:

https://github.com/topics/product-design
https://github.com/topics/guidelines
https://github.com/topics/howto
- https://github.com/pFarb/awesome-crypto-papers (off topic...)

**Comment by ItsExtra** [comment]
Mozilla [Open Leadership Series](https://mozilla.github.io/open-leadership-training-series/articles/readme/)

> This Open Leadership Training Series is designed to teach you best practices in “working open” – a way of working in which:
>   -  a community of people collaborate to make or build something amazing,
>   -  and any product or knowledge that the community of collaborators creates is shared widely and freely with the public on the world wide web.
> 
> This training is designed for those starting up or currently leading open projects– project leads, partners, or small groups of co-leaders responsible for project success and growth.

[8.1. Open Project Maintenance ](https://mozilla.github.io/open-leadership-training-series/articles/open-project-maintenance/open-project-maintenance/)
> Create a maintenance plan to prolong the life of your project and keep it accessible for your community and others.

LOOKS FUCKING BEAUTIFUL
